Cargo.toml

```diff
@@ -51,6 +52,9 @@ p256               = "0.13"
 proc-macro2        = "1"
 proptest           = "1"
 prost              = "0.13"
+pyth-sdk           = "0.8.0"
+pyth-wormhole      = { git = "https://github.com/pyth-network/pyth-crosschain", package = "pyth-wormhole-attester-sdk"}
```

Comment:
formatting here

```suggestion
pyth-wormhole      = { git = "https://github.com/pyth-network/pyth-crosschain", package = "pyth-wormhole-attester-sdk" }
```

Reply:
btw we need them to publish this package to crates.io, otherwise we can't publish ours

Reply:
it's also better to specify a commit hash when using a dependency from github